src/plugins/intel_cpu/src/nodes/executors/acl/acl_fullyconnected.cpp

```diff
@@ -0,0 +1,92 @@
+    aclTensorAttrs.hasLayoutTypeNHWC = memory.at(ARG_SRC)->getDescPtr()->hasLayoutType(LayoutType::nspc);
+    fullyConnectedLayerInfo.weights_trained_layout = getAclDataLayoutByMemoryDesc(memory.at(ARG_WEI)->getDescPtr());
+    fullyConnectedLayerInfo.transpose_weights = !attrs.weightsNonTransposed;
+    if (memory.at(ARG_SRC)->getPrecision() == ov::element::f16) {
+        fullyConnectedLayerInfo.fp_mixed_precision = true;
+    }
+
+    // Add postops
+    if (!postOps.empty() && postOps.size() == 1) {
+        if (const auto activation = std::dynamic_pointer_cast<ActivationPostOp>(postOps[0])) {
+            fullyConnectedLayerInfo.activation_info = getActivationLayerInfo(convertToEltwiseAlgorithm(activation->type()),
+                                                                             activation->alpha(),
+                                                                             activation->beta(),
+                                                                             activation->gamma());
+        }
+    }
+}
+
+bool ACLFullyConnectedExecutor::supports(const FCConfig &config) {
+    VERIFY(one_of(srcType(config), ov::element::f16, ov::element::f32), UNSUPPORTED_SRC_PRECISIONS);
+    VERIFY(postOpsNumbers(config) < 2,          UNSUPPORTED_NUMBER_OF_POSTOPS);
+    VERIFY(one_of(srcRank(config), 2U, 3U, 4U), UNSUPPORTED_SRC_RANK);
+    VERIFY(one_of(weiRank(config), 2U, 3U),     UNSUPPORTED_WEI_RANK);
+    return true;
+}
+
+arm_compute::Status ACLFullyConnectedExecutor::updateTensorsInfo(const ACLMemoryMap& acl_memory) {
+    auto wei_shape = acl_memory.at(ARG_WEI)->info()->tensor_shape();
+    if (wei_shape.num_dimensions() == 3U) {
+        acl_memory.at(ARG_WEI)->info()->set_tensor_shape({wei_shape[0] * wei_shape[1], wei_shape[2]});
+    }
+
+    auto src_shape = acl_memory.at(ARG_SRC)->info()->tensor_shape();
+    if (one_of(src_shape.num_dimensions(), 3U, 4U)) {
+        acl_memory.at(ARG_SRC)->info()->set_tensor_shape({
+            acl_memory.at(ARG_WEI)->info()->tensor_shape()[0],
+            src_shape.total_size() / acl_memory.at(ARG_WEI)->info()->tensor_shape()[0]});
+    }
+
+    if (one_of(acl_memory.at(ARG_DST)->info()->tensor_shape().num_dimensions(), 3U, 4U)) {
+        acl_memory.at(ARG_DST)->info()->set_tensor_shape({
+            acl_memory.at(ARG_WEI)->info()->tensor_shape()[1],
+            acl_memory.at(ARG_SRC)->info()->tensor_shape()[1]});
+    }
+
+    if (!fullyConnectedLayerInfo.transpose_weights) {
+        arm_compute::TensorShape temp_weights_shape = acl_memory.at(ARG_WEI)->info()->tensor_shape();
+        std::swap(temp_weights_shape[0], temp_weights_shape[1]);
+        acl_memory.at(ARG_WEI)->info()->set_tensor_shape(temp_weights_shape);
+    }
+
+    return arm_compute::NEFullyConnectedLayer::validate(
+            acl_memory.at(ARG_SRC)->info(),
+            acl_memory.at(ARG_WEI)->info(),
+            acl_memory.at(ARG_BIAS) ? acl_memory.at(ARG_BIAS)->info() : nullptr,
```

Comment:
Can we pass some kind of empty tensor with an empty tensor info?

Reply:
@EgorDuplensky corrected